packages/deprecation-crawler/src/lib/utils.ts

```diff
@@ -72,19 +74,19 @@ export function askToSkip<I>(
       return await crawlerProcess(d);
     }
 
-    const answer: { skip: string } = await prompt([
+    const answer: { proceed: boolean } = await prompt([
       {
-        type: 'select',
-        name: 'skip',
+        type: 'confirm',
```

Comment:
This change results in a question like this:


![image](https://user-images.githubusercontent.com/28659384/89150684-6917d380-d55f-11ea-99b7-edfd437d9f55.png)